src/AutoRest.CSharp/Mgmt/Decorator/Transformer/RearrangeParameterOrder.cs

```diff
@@ -24,7 +24,7 @@ public static void Update()
                             .Where(p => p.In == HttpParameterIn.Path)
                             .OrderBy(
                                 p => httpRequest.Path.IndexOf(
-                                    "{" + p.CSharpName() + "}",
+                                    "{" + p.Language.Default.SerializedName ?? p.Language.Default.Name + "}",
```

Comment:
`GetOriginalName`?

Reply:
Yes, we have that method, and I will change it here to use that.